spec/features/page_titles_spec.rb

```diff
@@ -0,0 +1,178 @@
+  let(:headers_resp) { { 'Accept' => mime_type, 'Content-Type' => mime_type } }
+
+  describe 'home page' do
+    it 'has title "Europeana Collections"' do
+      visit home_path(:en)
+      expect(page).to have_title('Europeana Collections', exact: true)
+    end
+  end
+
+  describe 'search page' do
+    it 'has title "Query - Search Results - Europeana Collections"' do
+      query = 'Beethoven'
+      visit search_path(:en, q: query)
+      expect(page).to have_title("#{query} - Search Results - Europeana Collections", exact: true)
+    end
+
+    it 'has title "Search Results - Europeana Collections"' do
+      visit search_path(:en, q: '')
+      expect(page).to have_title('Search Results - Europeana Collections', exact: true)
+    end
+  end
+
+  describe 'collections page' do
+    it 'has title "Collection name - Europeana Collections"' do
+      visit collection_path(:en, 'music')
+      expect(page).to have_title('Music - Europeana Collections', exact: true)
+    end
+  end
+
+  describe 'galleries page' do
+    it 'has title "Galleries - Europeana Collections"' do
+      visit galleries_path(:en)
+      expect(page).to have_title('Galleries - Europeana Collections', exact: true)
+    end
+    it 'has title "Gallery name - Galleries - Europeana Collections"' do
+      visit gallery_path(:en, 'fashion-dresses')
+      expect(page).to have_title('Fashion: dresses - Galleries - Europeana Collections', exact: true)
+    end
+  end
+
+  describe 'explore page' do
+    it 'has title "Explore name - Europeana Collections"' do
+      visit explore_newcontent_path(:en)
+      expect(page).to have_title('What\'s new? - Europeana Collections', exact: true)
+      visit explore_people_path(:en)
+      expect(page).to have_title('People - Europeana Collections', exact: true)
+    end
+  end
+
+  describe 'entity page' do
+    let(:body_entity) do
+      <<~EOM
+        {
+          "biographicalInformation": [
+            {
+              "@language": "en",
+              "@value": "..."
+            }
+          ],
+          "dateOfBirth": [ "1770-12-16" ],
+          "dateOfDeath": [ "1827-03-26" ],
+          "end": [ "1827-03-26" ],
+          "id": "http://data.europeana.eu/agent/base/146880",
+          "placeOfBirth": [
+            {
+              "@id": "http://dbpedia.org/resource/Electorate_of_Cologne"
+            },
+            {
+              "@id": "http://dbpedia.org/resource/Bonn"
+            }
+          ],
+          "placeOfDeath": {
+            "@id": "http://dbpedia.org/resource/Vienna"
+          },
+          "prefLabel": {
+            "en": "Ludwig van Beethoven"
+          },
+          "sameAs": [
+            "http://uz.dbpedia.org/resource/Ludwig_van_Beethoven"
+          ],
+          "type": "Agent"
+        }
+      EOM
+    end
+    it 'has title "Entity Name - Europeana Collections"' do
+      stub_request(:get, 'https://www.europeana.eu/api/entities/agent/base/1234?wskey=apidemo').
```

Comment:
This request should be stubbed like in spec/controllers/entities_controller_spec.rb:50 as now it assumes a entity API at europeana.eu but this is configurable.
Also rather than have the entity api hardcoded here, let's move it to a fixture same as what the entities controller is doing, that way we may be able to reuse it, or if possible add to the existing fixture and use that fixture here.